ethereum/spec/src/main/java/tech/pegasys/teku/spec/config/SpecConfigElectraImpl.java

```diff
@@ -195,7 +232,9 @@ public boolean equals(final Object o) {
         && maxDepositRequestsPerPayload == that.maxDepositRequestsPerPayload
         && maxWithdrawalRequestsPerPayload == that.maxWithdrawalRequestsPerPayload
         && maxPendingPartialsPerWithdrawalsSweep == that.maxPendingPartialsPerWithdrawalsSweep
-        && maxPendingDepositsPerEpoch == that.maxPendingDepositsPerEpoch;
+        && maxPendingDepositsPerEpoch == that.maxPendingDepositsPerEpoch
+        && maxBlobsPerBlockElectra == that.maxBlobsPerBlockElectra
+        && targetBlobsPerBlockElectra == that.targetBlobsPerBlockElectra;
```

Comment:
We're missing `blobSidecarSubnetCountElectra` here.

Reply:
And `maxBlobsPerBlockElectra`. Added both